File: add/codesnippet/CSharp/p-system.diagnostics.swi_2_1.cs
                // Display the SwitchLevelAttribute for the BooleanSwitch.
                Object[] attribs = typeof(BooleanSwitch).GetCustomAttributes(typeof(SwitchLevelAttribute), false);
                if (attribs.Length == 0)
                    Console.WriteLine("Error, couldn't find SwitchLevelAttribute on BooleanSwitch.");
                else
                    Console.WriteLine(((SwitchLevelAttribute)attribs[0]).SwitchLevelType.ToString());

File: add/codesnippet/VisualBasic/p-system.diagnostics.swi_2_1.vb
            ' Display the SwitchLevelAttribute for the BooleanSwitch.
            Dim attribs As [Object]() = GetType(BooleanSwitch).GetCustomAttributes(GetType(SwitchLevelAttribute), False)
            If attribs.Length = 0 Then
                Console.WriteLine("Error, couldn't find SwitchLevelAttribute on BooleanSwitch.")
            Else
                Console.WriteLine(CType(attribs(0), SwitchLevelAttribute).SwitchLevelType.ToString())
            End If